lightning/mydump/loader_test.go

```diff
@@ -11,17 +11,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package mydump_test
+package mydump
```

Comment:
prefer keeping the test package separate

Reply:
I change to package to avoid expose `fileInfo` for test, but end up with exposing it in restore. Thus I have changed back the package